source/renderer/app/components/wallet/WalletRestoreDialog.js

```diff
@@ -474,6 +505,59 @@ export default class WalletRestoreDialog extends Component<Props, State> {
           />
         )}
 
+        {!this.isCertificate() && this.isYoroi() && (
```

Comment:
@DeeJayElly here we should just use `this.isYoroi()`.

Reply:
Fixed